deploy-manage/users-roles/cluster-or-deployment-auth/controlling-access-at-document-field-level.md

```diff
@@ -109,6 +115,42 @@ POST /_security/role/dept_role
   ]
 }
 ```
+:::: 
+
+::::{tab-item} {{serverless-short}}
+:sync: serverless
+To configure document-level security (DLS), you create a custom role where you define the documents that this role grants access to, using the [QueryDSL](/explore-analyze/query-filter/languages/querydsl.md) syntax:
+
+1. Go to the **Custom Roles** page using the navigation menu or the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md).
```

Comment:
because serverless role management is done both in kibana and in elastic cloud, we should specify the surface

```suggestion
1. In {{kib}}, go to the **Custom Roles** page using the navigation menu or the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md).
```